### PR TITLE
VaibhavJ_Assignment_12

### PR DESCRIPTION
**Pull request by VaibhavJOct2020**
VaibhavJ_Assignment_12

### CODE REVIEW

**Comment by VaibhavJOct2020** [comment]
image

File: src/vaibhav/MinMaxNumber.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package vaibhav;
+
+public class MinMaxNumber {
+	
+	int minNumber(int[] arr) {		
+		int min=arr[0];
+		for (int index = 0 ;index < arr.length; index++){
```
Comment (harshvegada) [inline src/vaibhav/MinMaxNumber.java:7]: Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files

File: src/vaibhav/MinMaxNumber.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		for (int index = 0 ;index < arr.length; index++){			
+			if(arr[index] > max)
+				max=arr[index];
+			else continue;			
+		}		
+		return max;	
+		
+	}
+	
+	int avgAllNumber(int [] arr) {
```
Comment (harshvegada) [inline src/vaibhav/MinMaxNumber.java:26]: return type should be double

File: src/vaibhav/MinMaxNumber.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+		int avg=0;
+		for (int index = 0 ;index < arr.length; index++ ){
+			sum+=arr[index];			
+		}
+		avg= sum / arr.length;
+		return avg;		
+	}
+	
+	int avgMinMaxNumber(int min,int max) {
+		int ans=0;
```
Comment (harshvegada) [inline src/vaibhav/MinMaxNumber.java:37]: I think we have requirement where you need to find max & min number then you need to find avg of both, user will give us array, and you need to find min & max value. and then avg of both, I hope you got my point.

int avgOfMinMax = (getMinNumber(input)+getMaxNumber(input))/2

**Review by harshvegada — APPROVED** [review]
Over all looks good to me!
Logically everything is on place.
Good to merge!

**Pull request merged**